pmg/static/resources/css/style.scss

```diff
@@ -111,6 +111,9 @@ nav.top-links {
 
   .btn {
     margin-top: 6px;
+    padding: 5px 10px;
+    margin-left: 9px;
+    margin-right: -5px;
```

Comment:
Doesn't this break any buttons in the mega menu? I'd add a `.social-buttons` class to the LI and style on `.social-buttons .btn`